src/components/Compare/CompareStripedRows.tsx

```diff
@@ -11,8 +11,22 @@ function CompareStripedRows({
     compareB: string | boolean;
   }[];
 }) {
+  const getTableCell = (value: string | boolean) => {
+    if (value === true) {
+      return (
+        <div className="display-flex flex-justify-center">
+          <Check />
+        </div>
+      );
+    } else if (value === false) {
+      return null;
```

Comment:
the checkmarks might be an odd experience on screen readers...open to ideas on making this more screen-reader-friendly, e.g. having it say something explicit when there is a checkmark + absence of checkmark? cc @admoorgit

Reply:
I think we should add aria-labels to the checkboxes to say something to the effect of "[feature name] present at this location" / "[feature name] not present at this location"

Reply:
cool - waiting on adam to confirm language, will merge this and then we can have additional tweaks in a separate PR